--- a/usb_modeswitch.tcl
+++ b/usb_modeswitch.tcl
@@ -54,6 +54,9 @@
 		Log "Error: no config database found in /usr/share or /etc. Exiting"
 		SafeExit
 	}
+} else {
+	# Overriding from this place
+	set settings(dbdir_etc) /etc/usb_modeswitch.d
 }
 set bindir /usr/sbin
 
@@ -567,6 +570,8 @@
 				return {}
 			}
 			set configList [split $configList \n]
+			Log "Searching overriding entries named: $settings(dbdir_etc)/$config*"
+			append configList [glob -nocomplain $settings(dbdir_etc)/$config*]
 			set configList [lsearch -glob -all -inline $configList $config*]
 		} else {
 			set configList [glob -nocomplain $settings(dbdir)/$config*]
@@ -576,11 +581,17 @@
 	}
 	config {
 		if [file exists $settings(dbdir)/configPack.tar.gz] {
+		    # Overriding from /etc/
+		    if [file exists $settings(dbdir_etc)/$config] {
+			Log "Using overriden config $config from collection $settings(dbdir_etc)"
+			set settings(tmpConfig) $settings(dbdir_etc)/$config
+		    } else {
 			set settings(tmpConfig) /var/lib/usb_modeswitch/current_cfg
 			Log "Extracting config $config from collection $settings(dbdir)/configPack.tar.gz"
 			set wc [open $settings(tmpConfig) w]
 			puts -nonewline $wc [exec tar -xzOf $settings(dbdir)/configPack.tar.gz $config 2>/dev/null]
 			close $wc
+		    }
 		} else {
 			set settings(tmpConfig) $config
 		}
