Description: Disables the ustream-provided .rules file and comments
Origin:      vendor
Author:      Didier Raboud <didier@raboud.com>
Last-Update: 2009-08-31
Index: usb-modeswitch.git/80-usb_modeswitch.rules
===================================================================
--- usb-modeswitch.git.orig/80-usb_modeswitch.rules	2009-08-31 22:51:02.000000000 +0200
+++ usb-modeswitch.git/80-usb_modeswitch.rules	2009-08-31 22:55:53.000000000 +0200
@@ -1,3 +1,17 @@
+# Debian-specific:
+# 
+# Since udev 146, a modem-modeswitch program is provided and it does what
+# usb-modeswitch does, integrated in udev.
+#
+# Thus, this .rules file is now "opt-in" (disabled, but you can enable it
+# easily).
+#
+# If your device is not "modeswitched" automagically by udev (it should),
+# you can try to enable these usb-modeswitch rules, by commenting the line
+# below:
+GOTO="modeswitch_rules_end"
+# End of Debian-specific
+
 ACTION!="add", GOTO="modeswitch_rules_end"
 SUBSYSTEM!="usb", GOTO="modeswitch_rules_end"
 
