--- a/usb-modeswitch-upstart.conf
+++ b/usb-modeswitch-upstart.conf
@@ -1,5 +1,6 @@
 start on usb-modeswitch-upstart
 task
+env TMPDIR=/run
 script
 	exec /usr/sbin/usb_modeswitch_dispatcher --switch-upstart $UMS_PARAM
 end script
--- a/usb_modeswitch.sh
+++ b/usb_modeswitch.sh
@@ -85,6 +85,7 @@
  		elif [ -x /sbin/initctl ] && /sbin/initctl version 2>/dev/null | /bin/grep -q upstart; then # Test if upstart is running
 			exec initctl emit --no-wait usb-modeswitch-upstart UMS_PARAM=$1
 		else
+			export TMPDIR=/run
 			exec usb_modeswitch_dispatcher --switch-mode $1 &
 		fi
 		exit 0
--- a/usb_modeswitch@.service
+++ b/usb_modeswitch@.service
@@ -5,4 +5,5 @@
 Type=oneshot
 ExecStart=/usr/sbin/usb_modeswitch_dispatcher --switch-systemd %i
 #ExecStart=/bin/echo %i
+Environment="TMPDIR=/run"
 
